test/test.transporters.js

```diff
@@ -79,7 +90,9 @@ describe('Transporters', function() {
     var urlshortener = google.urlshortener('v1');
     var obj = { longUrl: 'http://google.com/' };
     urlshortener.url.insert({ resource: obj }, function(err, result) {
-      assert.deepEqual(err, { code: 500, message: 'There was an error!' });
+      assert(err instanceof Error);
+      assert.equal(err.code, 500);
+      assert.equal(err.message, 'There was an error!');
```

Comment:
Is it necessary...? We are testing if the object is instance of `Error`, that should implicitly test for presence of `stack`. What do you think...?